torch/_export/__init__.py

```diff
@@ -563,7 +563,17 @@ def export(
         preserve_module_call_signature=preserve_module_call_signature,
     )
 
+def disable_prexisiting_fake_mode(fn):
```

Comment:
Does this mean we are always using dynamo produced fake mode?

Reply:
yes! dynamo's fake_mode will be the single source of truth. This is easier to reason about and it also makes sure we won't mutate user provided fake_mode. The user provided input and model.named_parameters and buffers are still using outer fake_mode.